frontend/uddhar-frontend/src/coordinator/data/MapStat.jsx

```
@@ -0,0 +1,106 @@
+import axios from "axios";
+
+export const getCoordinates = async (location) => {
+  try {
+    const response = await axios.get(
+      `https://nominatim.openstreetmap.org/search?format=json&q=${location}`,
+    );
+    console.log("Coordinates response: ", response.data);
+    console.log("Location: ", location);
+    if (response.data?.length > 0) {
+      const result = {
+        lat: parseFloat(response.data[0].lat),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)